eGovPDX/portlandor
# POWR-1954 REFACTOR: Remove custom module alert handler

This PR removes the alert JavaScript and handler maintained within the custom `portland` module. This JS was removed in favor of the alerts.js within the theme.

**To Test:**
1. Navigate to http://powr-1954-portlandor.pantheonsite.io/
1. **Verify —>** The alerts still show up and there are no errors within the browser console pointing to a missing script or library
